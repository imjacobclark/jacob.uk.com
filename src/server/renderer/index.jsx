import App from '../../components/App.jsx';

export default dependencies => () => {
    const React = dependencies.getReact();
    const ReactDOMServer = dependencies.getReactDOMServer();

    return `
<html>
    <head>
    </head>
    <body>
        <main id="jacobclarkxyz__main">
            ${ReactDOMServer.renderToString(<App runtime="server"/>)}
        </main>
        <script src="client.bundle.js"></script>
    </body>
</html>`;
}