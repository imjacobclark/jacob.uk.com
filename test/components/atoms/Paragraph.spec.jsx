import React from 'react';
import renderer from 'react-test-renderer';
import Paragraph from '../../../src/components/atoms/Paragraph.jsx';

describe('Paragraph', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Paragraph>Hello I am Jacob Clark</Paragraph>).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
