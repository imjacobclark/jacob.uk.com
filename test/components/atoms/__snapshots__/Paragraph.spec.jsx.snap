// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph renders correctly 1`] = `
<p>
  Hello I am Jacob Clark
</p>
`;
